### ISSUE TITLE
PRAGMA table_info provides no output

### ISSUE THREAD

**Issue by mrigger** [cwida/duckdb#491]
Consider the following statements:

```sql
CREATE TABLE t0(c0 INT);
PRAGMA table_info('t0'); -- expected: {0|c0|INTEGER|false|NULL|false}, actual: {}
```
Unexpectedly, the `PRAGMA` provides no output. According to the documentation, it should be equivalent to the following, which actually provides an output:

```sql
SELECT * FROM pragma_table_info('t0'); -- {0|c0|INTEGER|false|NULL|false}
```
I found this issue based on commit bc9f086b9fcffb058763a6d1723da01127cd9100.

**Comment by hannesmuehleisen** [comment]
This also works for me, could you check again?

**Comment by mrigger** [comment]
This does not yet work for me (affecting both the release and debug builds).

Here is the output based on commit ef2286faef94a75d1ed02de0e86b804610da281e:

```
duckdb/build/release$ ./duckdb_cli 
SQLite version DuckDB ef2286fa
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite> CREATE TABLE t0(c0 INT);
sqlite> PRAGMA table_info('t0'); -- expected: {0|c0|INTEGER|false|NULL|false}, actual: {}
sqlite> SELECT * FROM pragma_table_info('t0'); -- {0|c0|INTEGER|false|NULL|false}
0|c0|INTEGER|false||false

duckdb/build/debug$ ./duckdb_cli 
SQLite version DuckDB ef2286fa
Enter ".help" for usage hints.
Connected to a transient in-memory database.
Use ".open FILENAME" to reopen on a persistent database.
sqlite> CREATE TABLE t0(c0 INT);
sqlite> PRAGMA table_info('t0'); -- expected: {0|c0|INTEGER|false|NULL|false}, actual: {}
sqlite> SELECT * FROM pragma_table_info('t0'); -- {0|c0|INTEGER|false|NULL|false}
0|c0|INTEGER|false||false
```

I'm using Ubuntu 19.04 and GCC (Ubuntu 8.3.0-6ubuntu1) 8.3.0.

**Comment by Mytherin** [comment]
Looks like this was specifically related to the SQLite API. The output was properly generated, but hidden because the SQLite shell only displays output of specific statement types. Should be fixed in b8a64e13c8c8a0dda3efd276c6f676503bb79052.

**Comment by mrigger** [comment]
Fixed by b8a64e13c8c8a0dda3efd276c6f676503bb79052.
